tests/test_middleware.py

```diff
@@ -57,8 +57,10 @@ def test_is_POST(self):
     @override_settings(OIDC_RP_CLIENT_ID='foo')
     @override_settings(OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS=120)
     @patch('mozilla_django_oidc.middleware.get_random_string')
-    def test_is_ajax(self, mock_random_string):
-        mock_random_string.return_value = 'examplestring'
+    @patch('mozilla_django_oidc.utils.get_random_string')
+    def test_is_ajax(self, mock_utils_random, mock_middleware_random):
+        mock_middleware_random.return_value = 'examplestring'
```

Comment:
For these and others, I suspect the mock of ``mozilla_django_oidc.middleware.get_random_string`` is no longer needed.

Reply:
I believe this is used to mock the `state` attribute in the views. Before this PR both `state` and `nonce` had the same value in tests. I will update the string in a following PR to make this clearer.

Reply:
No, you are right, I see where `examplestring` is being asserted later in the test, it still needs to be mocked. I was fooled by GitHub's context diff again. But, if we had a time machine, `examplestate` or `fakestate` would be better.